Source/cmake/WebKitCompilerFlags.cmake

```diff
@@ -139,6 +139,11 @@ endif ()
 if (COMPILER_IS_GCC_OR_CLANG)
     WEBKIT_APPEND_GLOBAL_COMPILER_FLAGS(-fno-strict-aliasing)
 
+    # Split debug information in ".debug_types" / ".debug_info" sections - this leads
+    # to a smaller overall size of the debug information, and avoids linker relocation
+    # errors on e.g. aarch64 (relocation R_AARCH64_ABS32 out of range: 4312197985 is not in [-2147483648, 4294967295])
+    WEBKIT_APPEND_GLOBAL_COMPILER_FLAGS(-fdebug-types-section)
```

Comment:
It doesn't matter much, because there's no way to override or undo -fdebug-types-section, but it's slightly nicer to use PREPEND rather than APPEND for this. APPEND is for stuff that's strictly required or the build is doomed.

Reply:
Fair enough! Thanks @mcatanzaro - will adapt.